cmake/common.cmake

```diff
@@ -307,6 +308,7 @@ function(add_piccolo_test)
       --label ${TEST_NAME} --perf-label ${PARSED_ARGS_PERF_LABEL}
       --snapshot-tx-interval 10000 ${PARSED_ARGS_ADDITIONAL_ARGS} -e
       ${ENCLAVE_TYPE} -t ${ENCLAVE_PLATFORM} ${NODES}
+    CONFIGURATIONS ${PARSED_ARGS_CONFIGURATIONS}
```

Comment:
I think this could set e2e_perf systematically, and not require the caller to do it, nor expose CONFIGURATIONS. Are there any performance tests that we do not want to group in that configuration?

Reply:
Done